core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -6050,6 +6050,14 @@ public String toString() {
               + "when Alluxio workers are required but not ready.")
           .setScope(Scope.CLIENT)
           .build();
+  public static final PropertyKey USER_HDFS_LISTSTATUS_MOUNTINFO_DISABLE =
+      booleanBuilder(Name.USER_HDFS_LISTSTATUS_MOUNTINFO_DISABLE)
+          .setDefaultValue(true)
+          .setIsHidden(true)
+          .setDescription("When a hdfs client listStatus a file, Whether not to use mount info")
```

Comment:
Let's change it to 

If enabled, the mount info will be excluded from the response when a HDFS client calls alluxio to list status on a directory.